test_financials.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+from unittest.mock import Mock
+from unittest.mock import patch
+import yfinance as yf
+
+class FinancialsTestCase(unittest.TestCase):
+    def setUp(self):
+        self.msft = yf.Ticker("MSFT")
```

Comment:
seems not to mock the data

Reply:
See the next line, line 9

Reply:
the line 8 yf.Ticker("MSFT") get data from web and in test function test if it empty, self.utils_mock seems not be used,

Reply:
The get_json call in utils is being mocked, so when get_financials() is called, it will indeed return empty.
Please review this to see how patch works: https://docs.python.org/3/library/unittest.mock.html